core/tests/protractor_utils/ProfilePage.js

```diff
@@ -70,6 +75,9 @@ var ProfilePage = function() {
   };
```

Comment:
Thanks for restoring `this.expectUserToHaveInterests`, but we still need a waitFor there before `return await interestElem.getText();`!